packages/hash/integration/src/tests/model/knowledge/linkEntity.model.test.ts

```diff
@@ -0,0 +1,209 @@
+import { getRequiredEnv } from "@hashintel/hash-backend-utils/environment";
+import { createGraphClient } from "@hashintel/hash-api/src/graph";
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+
+import {
+  EntityModel,
+  EntityTypeModel,
+  LinkEntityModel,
+  UserModel,
+} from "@hashintel/hash-api/src/model";
+import {
+  EntityTypeCreatorParams,
+  generateSystemEntityTypeSchema,
+  linkEntityTypeUri,
+} from "@hashintel/hash-api/src/model/util";
+import { generateTypeId } from "@hashintel/hash-shared/types";
+import { createTestUser } from "../../util";
+
+jest.setTimeout(60000);
+
+const logger = new Logger({
+  mode: "dev",
+  level: "debug",
+  serviceName: "integration-tests",
+});
+
+const graphApiHost = getRequiredEnv("HASH_GRAPH_API_HOST");
+const graphApiPort = parseInt(getRequiredEnv("HASH_GRAPH_API_PORT"), 10);
+
+const graphApi = createGraphClient(logger, {
+  host: graphApiHost,
+  port: graphApiPort,
+});
+
+describe("Link entity model class", () => {
+  let namespace: string;
+
+  let testUserModel: UserModel;
+  let testEntityTypeModel: EntityTypeModel;
+  let linkEntityTypeFriendModel: EntityTypeModel;
+  let linkEntityTypeAcquaintanceModel: EntityTypeModel;
+  let leftEntityModel: EntityModel;
+  let rightEntityFriendModel: EntityModel;
+  let rightEntityAcquaintanceModel: EntityModel;
+
+  const createEntityType = (
+    params: Omit<EntityTypeCreatorParams, "entityTypeId" | "actorId">,
+  ) => {
+    const entityTypeId = generateTypeId({
+      namespace,
+      kind: "entity-type",
+      title: params.title,
+    });
+    return EntityTypeModel.create(graphApi, {
+      ownedById: testUserModel.entityUuid,
+      schema: generateSystemEntityTypeSchema({
+        entityTypeId,
+        actorId: testUserModel.entityUuid,
+        ...params,
+      }),
+      actorId: testUserModel.entityUuid,
+    });
+  };
+
+  const _createEntity = (params: { entityTypeModel: EntityTypeModel }) =>
+    EntityModel.create(graphApi, {
+      ownedById: testUserModel.entityUuid,
+      properties: {},
+      actorId: testUserModel.entityUuid,
+      ...params,
+    });
```

Comment:
Is this being used?

Reply:
Not anymore, have removed in https://github.com/hashintel/hash/pull/1435/commits/eff1c1aa74f78c66b1db1b524f396897f1440174